src/MoonSharp.Interpreter/Options/FuzzySymbolMatchingBehaviour.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MoonSharp.Interpreter {
+
+	/// <summary>
+	/// A flag that controls if/how symbols (method, property, userdata) are fuzzily matched when they do not exist. Flags can be combined for multiple checks.
+	/// </summary>
+	[Flags]
+	public enum FuzzySymbolMatchingBehaviour
```

Comment:
**Your updated pull fixes this** Slight** nitpick, C# uses American English spelling variations so does MoonSharp (see src/MoonSharp.Interpreter/Serialization/SerializationExtensions.cs). I would say keep it as "Behavior" for consistency sake.

Reply:
I assumed since there was a class named ColonMatchingBehaviour also used in the options, the project used British English. If will adjust this.

Reply:
Good catch, actually. We should actually fix that to be US spelling.

However, the rest of the project is US spelling (`Initialize`, `Prioritize` etc.) Generally speaking, I think it wise that all code just standardise on US spelling conventions; particularly in the case of C# as .NET itself uses US spelling. I'm not too fussed about comments, commit messages etc.

P.S. Yes, I know that above I spelt _standardise_ with British spelling; I'm an Aussie 😉

Reply:
That's fair - looks like some inconsistent usage on our side; C# itself is based around american english, which could cause some weirdness in code; for example:

JsonSerializer serialiser = new JsonSerializer();
serialiser.Serialize(...)

With that said, since we are using Behaviour throughout the code right now, I'll create a separate issue to evaluate whether this should change - go ahead and follow the current pattern (Behaviour) for this.

Reply:
@Benjamin-Dobell  commented while I was commenting - I was going to suggest allowing this through and spawning a new issue for checking other inconsistencies. What do you think?

Reply:
Sorry, @LimpingNinja.

As you were going to suggest; just leave it for now.

There's not a single occurrence of US spelling "Behavior" in the project; so we should just go with what's already in place i.e. leave it as `FuzzySymbolMatchingBehaviour`.

We can sort out the UK/US spelling issues in a future commit - bit of a pain for API exposed identifiers, however I do think it's worth it for consistency purposes. Would need to be a 2.x release etc.

Reply:
Ackh. I committed my changes while you were speaking. Should I revert?

Reply:
I'll just be quiet and defer @LimpingNinja, rather than create any more confusion. Sorry!

Reply:
Apologies not needed! Always good to get second responses on these. I'm agreed.

Note for future visitors, here's an example:

JsonSerializer serialiser = new JsonSerializer();
serialiser.Serialize(...)

Similary for things like Color, Initialze, etc. While Behaviour will likely have no direct correlate (I think only WPF has behavior-type classes) it is good to make sure that the mindset is already there for future consistency.